native/cocos/platform/interfaces/modules/canvas/CanvasRenderingContext2D.cpp

```diff
@@ -233,10 +233,11 @@ void CanvasRenderingContext2D::setFont(const ccstd::string &font) {
             fontSizeStr = results[2].str();
             fontName = results[5].str();
         }
-
+        bool isItalic = font.find("italic", 0) != ccstd::string::npos || font.find("Italic", 0) != ccstd::string::npos;
```

Comment:
Is it possible to unify string processing with a single function？

Reply:
@bofeng-song will refactor related logic later on.